lib/state_manager/base.rb

```diff
@@ -165,7 +165,7 @@ def will_transition(from, to, event)
     def did_transition(from, to, event)
     end
 
-    def around_event(event, *args, &block)
+    def around_event(event, *args, **kwargs, &block)
       yield
     end
```

Comment:
wow. this repo has some bizarre code. i guess the method signature is `(event, *args, ...)` because that's what things are expecting, but it strikes me as odd that those aren't used (and that we bothered to name the passed in block? but that bit's probably an oversight)